dll/win32/shell32/CDefView.cpp

```diff
@@ -3503,6 +3571,46 @@ INT CDefView::_FindInsertableIndexFromPoint(POINT pt)
     return nCount;
 }
 
+void CDefView::_HandleStatusBarResize(int nWidth)
+{
+    LRESULT lResult;
+
+    if (m_isParentFolderSpecial)
+    {
+        int nPartArray[] = {-1};
+        m_pShellBrowser->SendControlMsg(FCW_STATUS, SB_SETPARTS, _countof(nPartArray), (LPARAM)nPartArray, &lResult);
+        return;
+    }
+
+    const int nFileSizePartLength = 125;
```

Comment:
```suggestion
    int nFileSizePartLength = 125;
```
My bad, I didn't see it was actually re-assigned down line 3593 😅